sno/ogr_import_source.py

```diff
@@ -387,6 +389,27 @@ def features(self):
         for ogr_feature in self._iter_ogr_features():
             yield self._ogr_feature_to_sno_feature(ogr_feature)
 
+    def get_features(self, row_pks, *, ignore_missing=False):
+        pk_field = self.primary_key
+
+        # TODO: does OGR have any better way of quoting these parameters?
```

Comment:
Not AFAIK, follows strict SQL formatting and expects you to have escaped already: https://gdal.org/user/ogr_sql_dialect.html